apps/wallet/src/screens/Dashboard/CollectionStack/index.tsx

```diff
@@ -0,0 +1,46 @@
+import type { FC } from 'react';
```

Comment:
We had a `stacks` dir for defining navigation system, could define these stacks inside the `stacks` dir, and keep this screen's structure

Reply:
> We had a `stacks` dir for defining navigation system, could define these stacks inside the `stacks` dir, and keep this screen's structure
okay, i'll move it to `stacks/Explorer`